### ISSUE TITLE
Allow other handlers apart from the file one

### ISSUE THREAD

**Issue by adinapoli** [ixmatus/snaplet-hslogger#2]
For example, syslog would be another popular choice.
A guess it would be enough to have a custom type which specifies the logging flavour:

``` haskell
data LoggingHandler = FileHandler | SysLogHandler
```

then pass it to the `initLogger` function and pattern match on it to determine which underlying `HsLogger` should be attached.

The choice shouldn't be limited to only one handler, so I guess such API is more probable:

``` haskell
initLogger :: [LoggingHandler] -> Initializer b v a
```

**Comment by ixmatus** [comment]
Yes, I didn't put too much thought in because I needed something quick but
we've got a lot of improvements we want to make, I'll keep your suggestion
in mind when we retool this (thank you).

If you make your own improvements before we get to it, I'll be happy to
accept a PR.

**Comment by adinapoli** [comment]
Fair enough ;) 
Thanks,

Alfredo
